Accelerated_Python_User_Guide/notebooks/Chapter_11_Distributed_Computing_cuPyNumeric.ipynb

```diff
@@ -53,7 +53,24 @@
    "source": [
     "## Basic installation\n",
     "\n",
-    "### Local linux server\n",
```

Comment:
Share the CUDA instructions between pypi and conda sections.

Reply:
https://github.com/NVIDIA/accelerated-computing-hub/pull/62/commits/b7e524f54338b4ee9a6c582bce2c30326f76fe56